### PR TITLE
Bug 2057566: ztp: add doc explaining mustonlyhave use when modifying list via PGT

### PR DESCRIPTION
**Pull request by pixelsoccupied**
/cc @lack

### CODE REVIEW

File: ztp/gitops-subscriptions/argocd/IssueWithCRGeneratedViaPolicy.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+## Issue With CR generated via Policy
+
+If you're using the default configurations and update a CR's list via PGT,  you may notice that the "final" CR generated doesn't match the Policy.
```
Comment (imiller0) [inline ztp/gitops-subscriptions/argocd/IssueWithCRGeneratedViaPolicy.md:3]:
```suggestion
If you're using the default configurations and update a CR's list via PolicyGenTemplate,  you may notice that the "final" CR generated doesn't match the Policy.
```

File: ztp/gitops-subscriptions/argocd/IssueWithCRGeneratedViaPolicy.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+## Issue With CR generated via Policy
```
Comment (imiller0) [inline ztp/gitops-subscriptions/argocd/IssueWithCRGeneratedViaPolicy.md:1]:
```suggestion
## Duplicated list entries when modifying Policy
```

File: ztp/gitops-subscriptions/argocd/IssueWithCRGeneratedViaPolicy.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+To work-around it just at `mustonlyhave` to the file under your PGT. E.g 
+
+```yaml
+- fileName: TunedPerformancePatch.yaml
+  complianceType: mustonlyhave
+```
+
+## Impact on CPU and system
+
+Adding `mustonlyhave` didn't have any impact on CPU consumption, but it's recommended to use `mustonlyhave` on an as-needed basis. When added keep an eye on the CPU consumption and any possible race conditions.
```
Comment (imiller0) [inline ztp/gitops-subscriptions/argocd/IssueWithCRGeneratedViaPolicy.md:68]: Do we expect that a race condition is possible?

Comment (lack) [reply ztp/gitops-subscriptions/argocd/IssueWithCRGeneratedViaPolicy.md:68]: I think "race condition" may be a short-hand for "ACM policy reconciliation loop" if any external entity adds annotations.

Comment (imiller0) [reply ztp/gitops-subscriptions/argocd/IssueWithCRGeneratedViaPolicy.md:68]: Can we clarify that line? It isn't a typical race condition. This is the case where there are two controllers ACM policy controller and a second controller both updating the same CR which causes an endless battle over the state of the CR.

Comment (pixelsoccupied) [reply ztp/gitops-subscriptions/argocd/IssueWithCRGeneratedViaPolicy.md]: Updated with a better expiation + added that helpful PromQL query

**Review by lack — APPROVED** [review]

**Pull request merged**